ai/states/play_state.py

```diff
@@ -47,7 +48,8 @@ def change_strategy(self, strategy_name: str, roles: Optional[Dict[Role, int]]=N
 
     def _is_mapping_valid(self, roles):
         for player_id in roles.values():
-            if player_id not in self.game_state.our_team.available_players.values():
+            if player_id not in self.game_state.our_team.available_players.keys():
+                self.logger.error("is not in available".format(player_id))
                 return False
```

Comment:
`self.logger.error('Player %d is not in available.', player_id)` ?

Reply:
I like the old way...

Reply:
mais il manque pas un `{}` ?

Reply:
yup

Reply:
Done